package.xml

```diff
@@ -16,19 +16,22 @@
   <email>Qianqian.Bu@microsoft.com</email>
   <active>yes</active>
  </lead>
- <date>2019-11-20</date>
- <time>13:00:13</time>
+ <date>2019-11-26</date>
+ <time>14:00:13</time>
  <version>
-  <release>1.0.2</release>
-  <api>1.0.2</api>
+  <release>1.0.3</release>
```

Comment:
It might be worthwhile to increase the minor version number (i.e. 1.1.0), because the important INI setting has been renamed and changed its meaning). Users are more likely to overlook that change in a patch revision.

Also, if there are any doubts regarding performance/behavior, it might be reasonable to first release this as beta version (i.e. 1.0.0beta1). If this is done, the stability element a few lines below also needs to be adjusted.

Reply:
Hi Christoph,

 Our team decide to change the design and go first with enableRedirect=on without fallback to see how customer will response for this. I've updated package version to 1.1.0beta1.  Thanks a lot for your suggestion.